src/game/creature.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "global/const.h"
+#include "global/types.h"
+
+#define CREATURE_SHOOT_RANGE SQUARE(WALL_L * 7) // = 51380224
+#define CREATURE_SHOT_DAMAGE 50
```

Comment:
This used to be `PEOPLE_SHOT_DAMAGE`. I don't see this new or old name used anywhere. Is it used somewhere outside the scope of this PR? Just want to make sure we didn't miss something.

Reply:
This is unused indeed, removed